### PR TITLE
Mod 3 Week 1 Assessment

### LINKED ISSUE #100: Weekly Assessment

### PR DESCRIPTION
**Pull request by memcmahon**
This assessment is based of of the learning goals on the [Curriculum Plan Doc](https://github.com/turingschool/launch-curriculum/blob/gh-pages/CurriculumPlan.md) (as of 5/31)

@rtillies We decided to have students do the assessment work in Replit.  Let us know if you were planning something different for the HTML/CSS lessons!

Closes #100

### CODE REVIEW

File: module3/assessments/week1-9715.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Week 3 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_Mod3Wk1_Assessment)
```
Comment (zoefarrell) [inline module3/assessments/week1-9715.md]: Yay already making it a separate repo so we don't have to do that the week before!!

The link is not working though, I think this is the correct link: https://github.com/turingschool-examples/Launch_Mod3Week1_Assessment

Comment (zoefarrell) [reply module3/assessments/week1-9715.md]: How do we want them to turn in the repl? Maybe they add the link into their GitHub repo somewhere?

Comment (memcmahon) [reply module3/assessments/week1-9715.md]: haha yep - I made the link before I made the repo 😆

Comment (zoefarrell) [reply module3/assessments/week1-9715.md]: The question mark is in a weird place in question 4, which was totally my doing 😆 How do you describe the difference between HTML and CSS, use an analogy or an example in your answer?

Comment (zoefarrell) [reply module3/assessments/week1-9715.md]: Just noticed it still says Week 3 Assessment in the assessment Repo

File: module3/assessments/week1-9715.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Week 3 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_Mod3Week1_Assessment)
```
Comment (zoefarrell) [inline module3/assessments/week1-9715.md:6]: Might be useful to put a link to the "solution" repl somewhere so we don't lose it if we want to make a modification at some point. Maybe another good place for an instructor note here with the link? https://replit.com/@Zoe-Farrell/Mod3Week1Assessment#

File: module3/assessments/week1-9715.md (hunk 1 above)
Comment (memcmahon) [reply module3/assessments/week1-9715.md]: updated for these last two comments!

File: module3/assessments/week1-9715.md (hunk 2 above)
Comment (memcmahon) [reply module3/assessments/week1-9715.md:6]: haha - yes I meant to include that 🙃

**Pull request merged**